vagga.yaml

```diff
@@ -2,20 +2,65 @@ containers:
   pip-freeze:
     setup:
       - !Alpine v3.4
+      - !BuildDeps
+        - postgresql-dev
       - !Py3Install
         - pip
         - 'Django >=1.9,<1.10'
         - 'gunicorn >=19.5,<19.6'
         - 'whitenoise >=3.2,<3.3'
         - 'django-environ >=0.4,<0.5'
+        - 'psycopg2 >=2.6,<2.7'
       - !Sh pip freeze > requirements.txt
   django:
     setup:
       - !Alpine v3.4
       - !PipConfig
         dependencies: true
+      - !Install
+        - libpq
       - !Py3Requirements requirements.txt
+  postgres:
+    setup:
+    - !Ubuntu xenial
+    - !EnsureDir /data
```

Comment:
I have an error without this 
`No such file or directory (os error 2) (source: exists, target: missing, mapped-root)`